src/site/stages/build/process-cms-exports/transformers/node-va_form.js

```diff
@@ -5,7 +5,15 @@ const {
   utcToEpochTime,
 } = require('./helpers');
 
+const moment = require('moment');
+
+function toUtc(dateString) {
+  const date = moment.utc(dateString);
+  return date.format('YYYY-MM-DD 12:00:00 [UTC]');
+}
+
 const transform = (entity, { ancestors }) => ({
```

Comment:
Should probably get 👀 from @ncksllvn and @zacharymorel here since they work with va_form's mostly! 🙂